bundles/org.openhab.binding.bluetooth.radoneye/src/main/java/org/openhab/binding/bluetooth/radoneye/internal/RadoneyeHandler.java

```
@@ -55,8 +55,8 @@ protected void updateChannels(int[] is) {
             data = RadoneyeDataParser.parseRd200Data(is);
             logger.debug("Parsed data: {}", data);
             Number radon = data.get(RadoneyeDataParser.RADON);
+            logger.debug("Parsed data radon number: {}", radon);
             if (radon != null) {
-                logger.debug("Parsed data radon: {}", String.format("[radon=%d Bq/m3]", radon));
```

This line fails silently.